tests/e2e/author-override.spec.js

```diff
@@ -0,0 +1,40 @@
+// import {test, expect} from '../tools/lib/test-utils.js';
+import {test, expect} from './tools/lib/test-utils.js';
+import {updatePost} from './tools/lib/post.js';
+
+const AUTHOR_NAME = 'Alternative Author';
+
+test.useAdminLoggedIn();
+
+// test.skip(({browserName}) => browserName === 'webkit', 'Skip on WebKit due to unsupported setting');
+
+test('Test Author override', async ({page, requestUtils}) => {
+  const newPost = await requestUtils.rest({
+    path: '/wp/v2/posts',
+    method: 'POST',
+    data: {
+      title: 'Test Author Override',
+      content: '<p>This is a test post</p>',
+      status: 'publish',
+      featured_media: 357,
+      categories: [1, 2, 3],
+    },
+  });
+  const editUrl = `./wp-admin/post.php?post=${newPost.id}&action=edit`;
+  const postUrl = newPost.link;
+
+  await page.goto(editUrl);
```

Comment:
On my local the test keeps failing at this point for Webkit
<img width="1335" height="583" alt="Screenshot 2025-10-31 at 14 55 25" src="https://github.com/user-attachments/assets/69e8dcd2-10d3-4968-94be-a773929e2ad6" />

Reply:
I think it happens because your OS.
Can you share me `Apple > About this Mac`?